mbed_devices/device.py

```diff
@@ -1,16 +1,14 @@
 """Device model definition."""
-from typing import NamedTuple
+from typing import NamedTuple, List
+import pathlib
 
 from mbed_targets import MbedTarget
-from mbed_devices.interface_firmware import InterfaceFirmware
 
 
 class Device(NamedTuple):
     """Definition of an Mbed Device."""
 
-    mount_point: str
-    identifier: str
-    interface_firmware: InterfaceFirmware
```

Comment:
This is something we get from the vendor ID. We should do a lookup using mbed targets e.g.
lookup(vendorId) -> name of the interface firmware (Jlink, Stlink, daplink)